src/main/webapp/i18n/en/tutorialGroups.json

```
@@ -0,0 +1,86 @@
+        "tutorialGroup": {
+            "deleted": "Tutorial group deleted"
+        },
+        "tutorialGroups": {
+            "tutorialGroupsButton": "Tutorial Groups"
+        },
+        "manageTutorialGroups": {
+            "title": "Tutorial Groups Management",
+            "tableCaption": "Tutorial Groups defined for this course",
+            "createButton": "Create Tutorial Group",
+            "columns": {
+                "id": "ID",
+                "title": "Title",
+                "teachingAssistant": "Teaching Assistant",
+                "capacity": "Capacity",
+                "registeredStudents": "Registered Students",
+                "isOnline": "Online/Offline",
+                "online": "Online",
+                "offline": "Offline",
+                "location": "Location",
+                "language": "Language",
+                "additionalInfo": "Additional Information"
+            },
+            "deleteButton": {
+                "question": "Are you sure to delete the Tutorial Group with title {{ title }}?",
+                "typeNameToConfirm": "Please enter the title of the tutorial group to confirm."
+            }
+        },
+        "createTutorialGroup": {
+            "title": "Create Tutorial Group"
+        },
+        "editTutorialGroup": {
+            "title": "Edit Tutorial Group"
+        },
+        "registeredStudents": {
+            "title": "Registered Students"
+        },
+        "tutorialGroupForm": {
+            "titleInput": {
+                "label": "Title",
+                "placeholder": "Group 1: Monday 12:00 - 13:00",
+                "requiredValidationError": "Title is required!",
+                "maxLengthValidationError": "Title can be max 255 characters long!"
+            },
+            "teachingAssistantInput": {
+                "label": "Teaching Assistant",
+                "placeholder": "Select a teaching assistant that will be responsible for this group.",
+                "loadingText": "Loading teaching assistants...",
+                "notFoundText": "No teaching assistants found in this course.",
+                "requiredValidationError": "A tutorial groups needs a teaching assistant assigned!",
+                "taHelp": "You can search by typing into the field."
+            },
+            "additionalInformationInput": {
+                "label": "Additional Information",
+                "placeholder": "Additional information about the tutorial group.",
+                "maxLengthValidationError": "Additional information is limited to 2000 characters!"
+            },
+            "locationInput": {
+                "labelOnline": "Meeting Link",
+                "labelOffline": "Meeting Place",
+                "placeholderOnline": "How can the students join the online meeting?",
+                "placeholderOffline": "How can the students find the room?",
+                "maxLengthValidationError": "Location is limited to 2000 characters!"
```

Please don't write any values into the translation file.
```suggestion
                "maxLengthValidationError": "Location is limited to {{limit}} characters!"
```